text/0000_require_minimal_ruby_version.md

```diff
@@ -0,0 +1,38 @@
+- Feature Name: mandatory_required_ruby_version
+- Start Date: 2020-04-13
+- RFC PR: 
+- Bundler Issue: 
+
+# Summary
+
+Calling `spec.required_ruby_version` in a gemspec should be mandatory.
+
+# Motivation
+
+My current problem is that many gems are not publishing their `required_ruby_version`.
+
+This means that a Gemfile/gemspec must know what is the maximum version of a gem for a given version of Ruby, instead of bundler figuring it out for us.
+
+It takes about one minute for the gem author to add the line. When the author doesn't, everyone that uses the gem, directly or indirectly, may have to figure out the maximum version and specify it in their own gemfiles.
```

Comment:
How do you determine which Ruby versions a gem is compatible with in a minute?

Reply:
Reworded.

Reply:
I think usually it is minimal version supported on CI for example (at least in my case). Anyway the gem could work in older versions as well.

:thinking: If I understand it wel, `required_ruby_version` was introduced to cover mainly syntax compatibility.